src/category_theory/discrete_category.lean

```diff
@@ -129,11 +130,13 @@ by tidy
 
 /-- Every functor `F` from a discrete category is naturally isomorphic (actually, equal) to
   `discrete.functor (F.obj)`. -/
+@[simp]
```

Comment:
Should this be `simp` or `simps`?

Reply:
I have verified that `simp` does the right thing here.